query_postal_codes_heatmaps_api.py

```diff
@@ -0,0 +1,104 @@
+from oauthlib.oauth2 import BackendApplicationClient
```

Comment:
You should sort the imports into 3 groups as per https://www.python.org/dev/peps/pep-0008/#imports. (A tool like https://github.com/PyCQA/isort can do it for you, although for one file it is probably overkill.)

Reply:
deee5e04f57ace2c72b3726e26d1fc3defdc5b02